controls/SV-204393.rb

```diff
@@ -1,35 +1,23 @@
+    The banner must be formatted in accordance with applicable #{input('org_name')} policy. Use the following verbiage for operating
     systems that can accommodate banners of 1300 characters:
-    "You are accessing a U.S. Government (USG) Information System (IS) that is provided for USG-authorized use only.
-    By using this IS (which includes any device attached to this IS), you consent to the following conditions:
-    -The USG routinely intercepts and monitors communications on this IS for purposes including, but not limited to,
-    penetration testing, COMSEC monitoring, network operations and defense, personnel misconduct (PM), law enforcement
-    (LE), and counterintelligence (CI) investigations.
-    -At any time, the USG may inspect and seize data stored on this IS.
-    -Communications using, or data stored on, this IS are not private, are subject to routine monitoring, interception,
-    and search, and may be disclosed or used for any USG-authorized purpose.
-    -This IS includes security measures (e.g., authentication and access controls) to protect USG interests--not for
-    your personal benefit or privacy.
-    -Notwithstanding the above, using this IS does not constitute consent to PM, LE or CI investigative searching or
-    monitoring of the content of privileged communications, or work product, related to personal representation or
-    services by attorneys, psychotherapists, or clergy, and their assistants. Such communications and work product are
-    private and confidential. See User Agreement for details."'
-  desc 'check', 'Verify the operating system displays the Standard Mandatory DoD Notice and Consent Banner before
+    \"#{input('banner_message_text_gui')}\" "
+  desc 'rationale', ''
+  desc 'check',"Verify the operating system displays the Standard Mandatory #{input('org_name')} Notice and Consent Banner before
     granting access to the operating system via a graphical user logon.
     Note: If the system does not have GNOME installed, this requirement is Not Applicable.
     Check to see if the operating system displays a banner at the logon screen with the following command:
     # grep banner-message-enable /etc/dconf/db/local.d/*
     banner-message-enable=true
-    If "banner-message-enable" is set to "false" or is missing, this is a finding.'
-  desc 'fix', 'Configure the operating system to display the Standard Mandatory DoD Notice and Consent Banner before
+    If \"banner-message-enable\" is set to \"false\" or is missing, this is a finding."
+  desc 'fix', "Configure the operating system to display the Standard Mandatory #{input('org_name')} Notice and Consent Banner before
     granting access to the system.
     Note: If the system does not have GNOME installed, this requirement is Not Applicable.
     Create a database to contain the system-wide graphical user logon settings (if it does not already exist) with the
```

Comment:
All of the text that you see in colors other than set for strings needs to be updated. You can escape the double quotes like we did on L19